### PR TITLE
Used standard blue color for all dialog boxes in Admin Panel

### LINKED ISSUE #368: Use standard blue color for all dialog boxes in Admin Panel
**Actual Behaviour**

Currently, a non-standard blue color is used in the dialog boxes.

**Expected Behaviour**

The standard blue color which is used throughout the site, should be used.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Akshat-Jain**
Fixes #368 

Changes: Used standard blue color for all dialog boxes in Admin Panel.

Surge Deployment Link: https://pr-397-fossasia-susi-accounts.surge.sh

### CODE REVIEW

**Comment by Akshat-Jain** [comment]
@PrP-11 Done. Please review. Thanks :-)

**Comment by Akshat-Jain** [comment]
@PrP-11 Please check now. Thanks :-)

**Comment by Akshat-Jain** [comment]
@akshatnitd Done. Please review. Thanks :-)

File: src/components/Admin/ListSkills/ListSkills.js [hunk 1]
```diff
@@ -363,7 +363,7 @@ class ListSkills extends React.Component {
       <FlatButton
         key={1}
         label="Change"
-        primary={true}
+        labelStyle={{ color: '#4285f4' }}
```
Comment (PrP-11) [inline src/components/Admin/ListSkills/ListSkills.js]: The standard blue color is still not added to constants folder. Please add it as a constant and then import it here

**Comment by Akshat-Jain** [comment]
@PrP-11 Done. Please review. Thanks :-)

**Pull request merged**